src/toil/lib/ec2nodes.py

```diff
@@ -246,8 +246,7 @@ def updateStaticEC2Instances():
         # See the License for the specific language governing permissions and
         # limitations under the License.
         from six import iteritems
-        from toil.lib.ec2nodes import InstanceType\n\n\n''',
-                            dictionary={'year': datetime.date.today().strftime("%Y")}))
+        from toil.lib.ec2nodes import InstanceType\n\n\n''').format(year=datetime.date.today().strftime("%Y"))[1:])
```

Comment:
Not really important,  but why `[1:]`?

Reply:
The string starts with a newline char so this just makes it neater when it writes the file.